userguide/storage.rst

```diff
@@ -1811,6 +1811,11 @@ click :guilabel:`+` to add that disk to the volume. Select *spare* in
 the :guilabel:`Volume layout` drop down. Click :guilabel:`Extend Volume`
 to add the hot spare.
 
+.. danger:: When adding a spare disk to an encrypted pool the
+   passphrase and recovery key are reset. Click
```

Comment:
I feel like the last part of that sentence needs emphasis.  ("the" is on the previous line, should be "**the"):
```suggestion
   passphrase and recovery key are reset**. Click
```